docs/Community/Committers.md

```diff
@@ -0,0 +1,50 @@
+column describes which technical areas each committer is most interested in, and
+therefore should usually be consulted for changes relating to that area.
+However, it is up to each committer to determine who should review which PR, and
+when to merge it. Remember that a PR must not be merged if a committer objects;
+instead, it should be brought up with the Community Governance Committee.
+
+| Name                  | GitHub Alias        | Area                           |
+|-----------------------|---------------------|--------------------------------|
+| Amaury Chamayou       | achamayou           | Build, CCF Integration         |
+| Anand Krishnamoorthi  | anakrish            | Debugging, SGX, EDL, Dev Tools |
+| Andrew Schwartzmeyer  | andschwa            | EDL, CMake, Dev Tools, Website |
+| Brett McLaren         | BRMcLaren           | Build, CMake, CI               |
+| Brian Telfer          | Britel              | TrustZone, Attestation         |
+| Simon Leet            | CodeMonkeyLeet      | SGX, APIs                      |
+| Dave Thaler           | dthaler             | TrustZone, APIs, Dev Tools     |
+| Emil Alexandru Stoica | EmilAlexandruStoica | Build, CMake, CI, Ansible      |
+| Akash Gupta           | gupta-ak            | SGX, Attestation, APIs         |
+| Hernan Gatta          | HernanGatta         | TrustZone                      |
+| Sergio Wong           | jazzybluesea        | Attestation, SGX               |
+| Jiri Appl             | jiria               | Attestation, TrustZone         |
+| John Kordich          | johnkord            | Build, CI, Dev Tools, Release  |
+| Jordan Hand           | jhand2              | Windows, Build, SGX            |
+| Xuejun Yang           | jxyang              | SGX                            |
+| Mike Brasher          | mikbras             | SGX, APIs, EDL                 |
+| Marius Oprin          | oprinmarius         | Build, CMake, CI, Ansible      |
+| Paul Allen            | paulcallen          | TrustZone                      |
+| Radhika Jandhyala     | radhikaj            | SGX, APIs, Website             |
+| Shruti Ratnam         | shruti25ratnam      | Attestation                    |
+| Cheng-mean Liu        | soccerGB            | Attestation, SGX               |
+| Bruce Campbell        | yakman2020          | Windows, SGX                   |
+
+Desired Status
+--------------
+
+As the project grows, in order to scale the management of the project, it is becoming increasingly necessary to delegate responsibility to domain-owners and improve the community's development workflow through automation. This will be handled through the use of [Prow](https://git.k8s.io/test-infra/prow).
+
+Commits should be fully gated through automated checks, with only a skeleton-crew of folks (a subset of the governance committee) able to directly merge code (and this power is only there for emergency measures, eg. if Prow has failed and needs to be restored).
```

Comment:
I think directly here means "without going through the automated checks/process". Line 11-12 does not give reviewers the power to bypass automated checks implemented by Prow, they presumably get discretion to merge only if those checks already pass.

Reply:
@dthaler you're absolutely correct. I've addressed this by preserving lines 7 - 14, and changing this section to a description of the current challenges (rather than a prescription for changes). I'll open that in a separate PR.